cmp/grid/GridModel.js

```diff
@@ -212,7 +214,10 @@ export class GridModel extends HoistModel {
      * @param {(string|string[])} [c.groupBy] - Column ID(s) by which to do full-width row grouping.
      * @param {boolean} [c.showGroupRowCounts] - true (default) to show a count of group member
      *      rows within each full-width group row.
-     * @param {string} [c.sizingMode] - one of large, standard, compact, tiny
+     * @param {string} [c.sizingMode] - one of large, standard, compact, tiny. If undefined, will
+     *      default and bind to `XH.sizingMode`.
+     * @param {boolean} [c.autosizeOnSizingModeChange] - trigger autosize whenever the sizing mode
```

Comment:
this seems quite specific -- could we think about working this into the existing `autosizeOptions` bundle?